### ISSUE TITLE
0330IbnCazizSijistani

### ISSUE THREAD

**Issue by pverkind** [OpenITI/Annotation#2449]
**OLD URI:** 0330IbnCazizSijistani
**NEW URI:** 0330IbnCuzayrSijistani
**REASON:** Two URIs for the same person: correct is ʿUzayr see https://viaf.org/viaf/23706113/

**NOTE:** Please, paste the *old* URI into the header. If you think the death date or the author's name should be changed in all URIs of that author, please fill in the author URI only (e.g., 0255Jahiz). If you think something needs to change in the title of a book, in all versions, please fill in the book URI only (e.g., 0255Jahiz.Hayawan). Finally, if a URI change relates only to a specific version of the book, fill in the Version URI (e.g., 0255Jahiz.Hayawan.Shamela0000001-ara1).

**Comment by aslishah** [comment]
Yes, I also wanted to flag this. Although I can't say which version of the _Gharib al-Quran_ should be made secondary.
